src/python/visclaw/data.py

```diff
@@ -621,12 +622,17 @@ def __init__(self, name, figno, fig_type, plotdata):
         self._show = True
         self.add_attribute('plotaxes_dict', {})
         self.add_attribute('type',fig_type)   # = 'each_frame' or 'each_run' or 'each_gauge'
+        self.add_attribute('use_for_kml',False)
+        self.add_attribute('kml_dpi',200)
+        self.add_attribute('kml_xlimits',None)
```

Comment:
But would you want to do this using a different interface than what is already available?

In response to Donna, you do have a point, it is actually non-trivial to get the axes limits from the automatically generated bounds (you would essential have to fake plot once and grab them afterwards).  That being said, you could always look at the solution object itself when actually plotting and use those as the bounding limits.  My view on the latter problem is that if the user wants to place the axes specifically they should not specify `auto` anyway.  Also, if there are multiple axes per figure we could raise an exception and not allow that to occur.  I think I would still prefer this route, providing feedback but staying with the same interface, than adding similar options but to a different level of the data hierarchy.